Traceback (most recent call last):
  File "E:\Learning_material\junior\AI\5_FinalLab\FinalLab_ai\compare.py", line 219, in <module>
    main()
  File "E:\Learning_material\junior\AI\5_FinalLab\FinalLab_ai\compare.py", line 203, in main
    train_loader, val_loader = load_data()
                               ^^^^^^^^^^^
  File "E:\Learning_material\junior\AI\5_FinalLab\FinalLab_ai\compare.py", line 51, in load_data
    with open(TRAIN_FILE, 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^
OSError: [Errno 22] Invalid argument: 'E:\\Learning_material\\junior\\AI\x05_FinalLab\\dataset\\train.txt'
